react/tic-tac-toe/src/index.js

```diff
@@ -1,17 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
-
+import { Provider } from 'react-redux';
+import { applyMiddleware, compose, createStore, combineReducers as CR } from 'redux';
+import { reducer as formReducer } from 'redux-form';
+import { fetchMiddleware } from 'redux-recompose';
 import App from './app';
 import './config/i18n';
 import './scss/application.scss';
+import apiReducer from "./redux/reducer";
 import { register } from './serviceWorker';
+import thunk from 'redux-thunk';
+
+const reducers = {
+  form: formReducer,
+  tictactoe: apiReducer,
+};
+const combineReducers = CR;
```

Comment:
why do you do this? I'd import it with it's default name and delete this

Reply:
:white_check_mark: